plugins/3MFReader/ThreeMFWorkspaceReader.py

```diff
@@ -229,6 +227,7 @@ def preRead(self, file_name, show_dialog=True, *args, **kwargs):
         # Read definition containers
         #
         machine_definition_id = None
+        updatable_machines = []
```

Comment:
I think you should rename this to updatable_machine_names (since the list doesn't actually contain machines). Also, it's missing typing.